### PR TITLE
4.x Adding types to the shell package

### PR DESCRIPTION
**Pull request by burzum**
Refs https://github.com/cakephp/cakephp/issues/11935

### CODE REVIEW

**Review by markstory — COMMENTED** [review]
You might also want to add `declare(strict_types=1)` to find more typing related issues.

**Review by ADmad — COMMENTED** [review]
Strict typing should be enabled for the related test case files too.

File: src/Shell/CacheShell.php [hunk 1]
```diff
@@ -1,4 +1,5 @@
 <?php
+declare(strict_types = 1);
```
Comment (saeideng) [inline src/Shell/CacheShell.php]: I think extra spaces should remove 
`_typeses = 1)` = >`_typeses=1)`

Comment (saeideng) [reply src/Shell/CacheShell.php]: https://github.com/cakephp/cakephp-codesniffer/pull/205

**Pull request merged**